docs/generated/packages/nx/documents/init.md

```diff
@@ -5,7 +5,7 @@ description: 'Adds Nx to any type of workspace. It installs nx, creates an nx.js
 
 # init
 
-Adds Nx to any type of workspace. It installs nx, creates an nx.json configuration file and optionally sets up distributed caching. For more info, check https://nx.dev/recipes/adopting-nx.
+Adds Nx to any type of workspace. It installs nx, creates an nx.json configuration file and optionally sets up distributed caching. For more info, check [https://nx.dev/recipes/adopting-nx](https://nx.dev/recipes/adopting-nx).
```

Comment:
Hey, a few things:
- This is generated documentation, can't just change it here
- This link shows up in the help text for `nx init`, and the text here is pulled from it. Markdown doesnt render in the terminal, so we can't use a markdown link here.

Reply:
Got it! Didn't know that it also appeared in the terminal that way ✌️